src/Dialogs/BulkRenamer/RenamerDialog.vala

```diff
@@ -102,8 +102,9 @@ public class Files.RenamerDialog : Gtk.Dialog {
         suffix_menumodel.append (_("Fixed Text"), Action.print_detailed_name ("renamer.add-text", suffix_var));
 
         prefix_button = new Gtk.MenuButton () {
-            image = new Gtk.Image.from_icon_name ("add-symbolic", Gtk.IconSize.BUTTON),
-            tooltip_text = _("Add Prefix"),
+            always_show_image = true,
+            image = new Gtk.Image.from_icon_name ("list-add-symbolic", Gtk.IconSize.BUTTON),
+            label = _("Add Prefix"),
```

Comment:
I feel like this is such an abstract idea that just image buttons probably aren't sufficient here